src/backends/odbc/session.cpp

```diff
@@ -387,6 +399,8 @@ odbc_session_backend::get_database_product() const
         product_ = prod_postgresql;
     else if (strcmp(product_name, "SQLite") == 0)
         product_ = prod_sqlite;
+    else if (strstr(product_name, "DB2") == product_name) // "DB2/LINUXX8664"
```

Comment:
Just curious, do you know if upper-case `DB2` is guaranteed across DB2 editions/versions?